### PR TITLE
Fix strong reference

### PR DESCRIPTION
**Pull request by zenangst**
With the help of the Magician aka @RamonGilabert, we solved a huge memory leak that was caused by a strong reference to the lightbox controller.

This PR also introduces and fixes som swift lint warnings.

### CODE REVIEW

File: Source/LightboxController.swift [hunk 1]
```diff
@@ -145,8 +145,8 @@ public class LightboxController: UIViewController {
   var pageViews = [PageView]()
   var statusBarHidden = false
 
-  private let initialImages: [LightboxImage]
-  private let initialPage: Int
+  private var initialImages: [LightboxImage]
```
Comment (vadymmarkov) [inline Source/LightboxController.swift]: Why `var`?

Comment (zenangst) [inline Source/LightboxController.swift]: oh crap, my bad!

Comment (zenangst) [inline Source/LightboxController.swift]: Fixed!

**Comment by zenangst** [comment]
@vadymmarkov fixed! 😁

File: Source/Library/LightboxTransition.swift [hunk 2]
```diff
@@ -22,7 +22,7 @@ class LightboxTransition: UIPercentDrivenInteractiveTransition {
     }
   }
 
-  var lightboxController: LightboxController?
+  weak var lightboxController: LightboxController?
```
Comment (onmyway133) [inline Source/Library/LightboxTransition.swift]: so this is the culprit?

Comment (zenangst) [inline Source/Library/LightboxTransition.swift]: Oh yes, it was to blame! 😁

**Pull request merged**